count_server/web/templates/layout/app.html.eex

```diff
@@ -30,6 +30,7 @@
       </main>
 
     </div> <!-- /container -->
+    <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
I wonder if you want to npm install and include jquery that way rather than in there as a script tag. I honestly don't know the answer, am just wondering it...